src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/MainPageTests.java

```diff
@@ -0,0 +1,475 @@
+
+    private String name;
+    private String className;
+
+    PageElements(String name, String className) {
+      this.name = name;
+      this.className = className;
+    }
+  }
+
+  // CCT01
+  @Test(groups = {"MercuryCuratedMainPageTests_001", "MercuryCuratedMainPageTests", "Mercury"})
+  public void MercuryCuratedMainPageTests_001_CheckElementsVisibilityElementsOrderAndRootPath() {
+    CuratedMainPagePageObject cc = new CuratedMainPagePageObject(driver);
+    wikiURL = urlBuilder.getUrlForWiki(MercuryWikis.MERCURY_CC);
+    cc.openMercuryArticleByName(wikiURL, MercurySubpages.CC_MAIN_PAGE);
+
+    boolean result = cc.isUrlPathEqualTo(ROOT_PATH);
+    PageObjectLogging.log(
+        "Current URL",
+        "is set on " + ROOT_PATH,
+        "is not set on " + ROOT_PATH,
+        result
+    );
+
+    result = cc.isMobileTopLeaderboardVisible();
+    PageObjectLogging.log(
+        PageElements.TOP_LEADERBOARD.name,
+        MercuryMessages.VISIBLE_MSG,
+        MercuryMessages.INVISIBLE_MSG,
+        result
+    );
+
+    result = cc.isArticleTitleVisible();
+    PageObjectLogging.log(
+        PageElements.ARTICLE_TITLE.name,
+        MercuryMessages.VISIBLE_MSG,
+        MercuryMessages.INVISIBLE_MSG,
+        result
+    );
+
+    result = cc.isWikiaStatsContainerVisible();
+    PageObjectLogging.log(
+        PageElements.WIKIA_STATS.name,
+        MercuryMessages.VISIBLE_MSG,
+        MercuryMessages.INVISIBLE_MSG,
+        result
+    );
+
+    result = cc.isFeaturedContentVisible();
+    PageObjectLogging.log(
+        PageElements.FEATURED_CONTENT.name,
+        MercuryMessages.VISIBLE_MSG,
+        MercuryMessages.INVISIBLE_MSG,
+        result
+    );
+
+    result = cc.isCuratedContentVisible();
+    PageObjectLogging.log(
+        PageElements.CURATED_CONTENT.name,
+        MercuryMessages.VISIBLE_MSG,
+        MercuryMessages.INVISIBLE_MSG,
+        result
+    );
+
+    result = cc.isMobileInContentVisible();
+    PageObjectLogging.log(
+        PageElements.IN_CONTENT.name,
+        MercuryMessages.VISIBLE_MSG,
+        MercuryMessages.INVISIBLE_MSG,
+        result
+    );
+
+    result = cc.isTrendingArticlesVisible();
+    PageObjectLogging.log(
+        PageElements.TRENDING_ARTICLES.name,
+        MercuryMessages.VISIBLE_MSG,
+        MercuryMessages.INVISIBLE_MSG,
+        result
+    );
+
+    result = cc.isTrendingVideosVisible();
+    PageObjectLogging.log(
+        PageElements.TRENDING_VIDEOS.name,
+        MercuryMessages.VISIBLE_MSG,
+        MercuryMessages.INVISIBLE_MSG,
+        result
+    );
+
+    result = cc.isMobilePrefooterVisible();
+    PageObjectLogging.log(
+        PageElements.PREFOOTER.name,
+        MercuryMessages.VISIBLE_MSG,
+        MercuryMessages.INVISIBLE_MSG,
+        result
+    );
+
+    int lastPosition = 0;
+    String lastElement = "top";
+
+    for (PageElements element : PageElements.values()) {
+      int newPosition = cc.getElementHeight(element.className);
```

Comment:
also you can move this logic to page object.
For instance:
method inside PO: getElementsOrder();
assertion in test: assertEquals(orderOnPage, excpectedOrder)